v4_MIGRATION_GUIDE.md

```diff
@@ -536,7 +536,99 @@ However, `requestOneTimePayment` and `requestBillingAgreement` have been depreca
 
 ## Samsung Pay
 
-Samsung Pay is not yet supported in v4.
+The SamsungPay feature is now supported in a single dependency:
+
+```groovy
+dependencies {
+  implementation 'com.braintreepayments.api:samsung-pay:4.2.0'
+}
+```
+
+To use the feature, instantiate a `SamsungPayClient`:
+
+```java
+package com.my.app;
+
+public class SamsungPayActivity extends AppCompatActivity implements SamsungPayListener {
+    
+  private BraintreeClient braintreeClient;
+  private SamsungPayClient samsungPayClient;
+
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    braintreeClient = new BraintreeClient(this, "TOKENIZATION_KEY_OR_CLIENT_TOKEN");
+    samsungPayClient = new SamsungPayClient(braintreeClient);
+  }
+
+  private void checkIfSamsungPayIsAvailable() {
+    samsungPayClient.isReadyToPay((isReadyToPay, error) -> {
+      if (isReadyToPay) {
+        // Samsung Pay is available 
```

Comment:
The merchant should call `launchSamsungPay()` here right?

Reply:
Good call yeah I think they may want to display the button here. We may want to update other docs Google Pay says the same thing.